distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -142,12 +142,10 @@ grant codeBase "file:/jetty-server/catalog-core-directorymonitor/catalog-securit
     permission java.io.FilePermission "${ddf.home.perm}etc${/}system.properties", "read,write";
 }
 
+// Any command executed via SSH will use other bundles that have their file access properly secured
+// so allowing karaf's SSH bundles access to all files won't create any security vulnerabilities.
 grant codeBase "file:/org.apache.karaf.shell.ssh/org.apache.sshd.core" {
-    permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}-", "read";
-    permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}server${/}-", "read";
-    permission java.io.FilePermission "${ddf.home.perm}etc${/}host.key", "read,write";
-    permission java.io.FilePermission "${ddf.home.perm}etc${/}branding-ssh.properties", "read";
-    permission java.io.FilePermission "${ddf.home.perm}etc${/}org.apache.karaf.shell.cfg", "read";
+    permission java.io.FilePermission "<<ALL FILES>>", "read,write";
```

Comment:
@stustison Should we be giving more than file permissions to these 2 as per our offline discussion?

Reply:
Let's just leave it at file access for now and see how it goes. This is pretty much what I was expecting to see.